client/pages/search-list.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import NotFound from './not-found';
+
+export default class SearchList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      trails: [],
+      isSubmitted: false
+    };
+  }
+
+  componentDidMount() {
+    fetch('/api/searched-trails?trailName=' + encodeURIComponent(`${this.props.searchKeyword}`))
+      .then(res => res.json())
+      .then(trails => {
+        this.setState({
+          isSubmitted: true,
+          trails: trails
+        });
+      });
+  }
+
+  render() {
+    if (this.state.trails.length === 0) {
+      return <NotFound/>;
+    }
+    return (
+        <>
+        <div className="row justify-center" >
+          <h3 className="add-trail-title">Search Result</h3>
+        </div >
+
+        <div>
+          {
+            this.state.trails.map(trail => {
+              return (
+                <div key={trail.trailId} className="row trail-entry">
+                  <Trail trail={trail} />
+                </div>
+              );
+            })
+          }
+        </div>
+        <div className="add-button">
+          <label htmlFor='img-upload' className="image-upload-button">
+            <a href="#submit"><i className="fas fa-plus-circle plus-button"></i></a>
+            <a href="#submit"><p className="click-to-add">CLICK TO ADD</p></a>
+          </label>
+          <input id="img-upload" type="file" />
```

Comment:
I'm not sure what this is for right now, but it seems out of place. inputs and buttons for forms should go in a form element.